staging/src/kubevirt.io/client-go/kubecli/websocket.go

```diff
@@ -55,57 +54,70 @@ func Dial(address string, tlsConfig *tls.Config) (*websocket.Conn, *http.Respons
 	return dialer.Dial(address, nil)
 }
 
-func Copy(dst *websocket.Conn, src *websocket.Conn) (written int64, err error) {
-	return copy(&binaryWriter{conn: dst}, &binaryReader{conn: src})
+func Copy(dst *websocket.Conn, src *websocket.Conn) (int64, error) {
+	return io.Copy(dst.UnderlyingConn(), src.UnderlyingConn())
 }
 
 func CopyFrom(dst io.Writer, src *websocket.Conn) (written int64, err error) {
-	return copy(dst, &binaryReader{conn: src})
+	return io.Copy(dst, &binaryReader{conn: src})
 }
 
 func CopyTo(dst *websocket.Conn, src io.Reader) (written int64, err error) {
-	return copy(&binaryWriter{conn: dst}, src)
-}
-
-func copy(dst io.Writer, src io.Reader) (written int64, err error) {
-	// our websocket package has an issue where it truncates messages
-	// when the message+header is greater than the buffer size we allocate.
-	// thus, we copy in chunks of WebsocketMessageBufferSize-wsFrameHeaderSize
-	buf := make([]byte, WebsocketMessageBufferSize-wsFrameHeaderSize)
-	return io.CopyBuffer(dst, src, buf)
+	return io.Copy(&binaryWriter{conn: dst}, src)
 }
 
 type binaryWriter struct {
-	conn *websocket.Conn
+	conn  *websocket.Conn
+	write int
 }
 
 func (s *binaryWriter) Write(p []byte) (int, error) {
+	s.write++
 	w, err := s.conn.NextWriter(websocket.BinaryMessage)
 	if err != nil {
 		return 0, convert(err)
 	}
 	defer w.Close()
-	return w.Write(p)
+	n, err := w.Write(p)
+	return n, err
 }
 
 type binaryReader struct {
-	conn *websocket.Conn
+	conn   *websocket.Conn
+	read   int
```

Comment:
do you need this int? i'm guessing it was used for debug

Reply:
yep debugging leftover.